src/qibosoq/rfsoc_server.py

```diff
@@ -1,21 +1,25 @@
 """Qibosoq server for qibolab-qick integration.
 
-Supports the following FPGA:
+Tested on the following FPGA:
     * RFSoc4x2
+    * ZCU111
 """
 
+import json
 import logging
 import os
-import pickle
 import socket
+import traceback
 from socketserver import BaseRequestHandler, TCPServer
 
 from qick import QickSoc
 
-from qibosoq.qick_programs import ExecutePulseSequence, ExecuteSingleSweep
+import qibosoq.configuration as qibosoq_cfg
```

Comment:
```suggestion
import qibosoq.configuration as cfg
```

You're in `qibosoq`, `cfg` should be good enough ;)